.github/workflows/python-publish.yml

```diff
@@ -18,15 +18,9 @@ jobs:
     - name: Set up Python
       uses: actions/setup-python@v2
       with:
-        python-version: '3.x'
-    - name: Install dependencies
-      run: |
-        python -m pip install --upgrade pip
-        pip install twine build
-    - name: Build and publish
-      env:
-        TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
-        TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
-      run: |
-        python -m build
-        twine upload dist/*
+        python-version: '3.10'
```

Comment:
Maybe double quotes to be consistent with the rest of python versions defined in the `yml`. I see it has changed to string in the pre-commit PR